File: zuzubibi/2월/2월 19일/최댓값과 최솟값.py
def solution(s):
    a = [int(i) for i in s.split()]
    return ' '.join([str(min(a)), str(max(a))])

def anotherSolutino(s):
    s = list(map(int, s.split()))
    return str(min(s)) + ' ' + str(max(s))

print(solution("1 2 3 4")) # "1 4"
print(solution("-1 -2 -3 -4")) # "-4 -1"
print(anotherSolutino("-1 -1")) #"-1 -1"

File: yht0827/2월/2월 19일/최댓값과 최솟값.java
class Solution {
	public String solution(String s) {
		String answer = "";

		String[] numbers = s.split(" ");
		int min = Integer.MAX_VALUE;
		int max = Integer.MIN_VALUE;

		for (int i = 0; i < numbers.length; i++) {
			int number = Integer.parseInt(numbers[i]);

			min = Math.min(min, number);
			max = Math.max(max, number);
		}

		answer = min + " " + max;
		return answer;
	}
}